docs/tables/aws_backup_framework.md

```diff
@@ -0,0 +1,169 @@
+# Table: aws_backup_framework
+
+AWS Backup Framework is a collection of controls that you can use to evaluate your backup practices.
+The AWS Backup Framework will then evaluate whether your backup practices comply with your policies and highlights which
+resources are not yet in compliance.
+
+## Examples
+
+### Basic Info
```

Comment:
We have a problem here that other documents use this as capitalised. 
I copied from those documents. If we change this, it will make the document inconsistent to other documents.